prow/config.yaml

```diff
@@ -13847,6 +14042,82 @@ periodics:
       secret:
         secretName: velodrome-influxdb
 
+- name: periodic-kubernetes-bazel-build-1-10
+  interval: 6h
+  agent: kubernetes
+  labels:
+    preset-service-account: true
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/bazelbuild:v20180201-0184a54dc-0.8.1
+      args:
+      - "--clean"
+      - "--git-cache=/root/.cache/git"
+      - "--job=$(JOB_NAME)"
+      - "--repo=k8s.io/kubernetes=release-1.10"
+      - "--service-account=/etc/service-account/service-account.json"
+      - "--upload=gs://kubernetes-jenkins/logs"
+      - "--" # end bootstrap args, scenario args below
+      - "--build=//... -//vendor/..."
+      - "--release=//build/release-tars"
+      env:
+      - name: TEST_TMPDIR
+        value: /root/.cache/bazel
+      # Bazel needs privileged mode in order to sandbox builds.
+      securityContext:
+        privileged: true
+      volumeMounts:
+      - name: cache-ssd
+        mountPath: /root/.cache
+      ports:
+      - containerPort: 9999
+        hostPort: 9999
+      resources:
+        requests:
+          memory: "6Gi"
+    volumes:
+    - name: cache-ssd
+      hostPath:
+        path: /mnt/disks/ssd0
+  run_after_success:
```

Comment:
this also shouldn't be run_after_success anymore, and the same comments about caching / the images apply. these changed since the PR was created, 1.10/master should use the newer images